### ISSUE TITLE
Download SVG on assets page 404s in non-English languages

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#12267]
### Describe the bug

The Download SVG link on is broken and currently 404s in any language except English

- The correct path: https://ethereum.org/assets/svgs/eth-diamond-black.svg
- Path in non-English languages (404): https://ethereum.org/fr/assets/svgs/eth-diamond-black.svg

### To reproduce

1. Go to 'https://ethereum.org/assets'
2. Pick any language
3. Click on 'Download (SVG)'

### Expected behavior

Shouldn't 404

### Additional context

@pettinarip was working on a fix to re-add 'en' paths to our canonical URLs. I'm assuming these download SVG buttons will also 404 in English when we make this change.

### Would you like to work on this issue?

- [ ] Yes
- [X] No

**Comment by kamuik16** [comment]
I think the problem here is the path definition. 
When English is selected as a language the path to any svg i.e. is `https://ethereum.org/assets/svgs/eth-diamond-glyph.svg` while if any other language is selected the svg's path link adds the language abbreviation in it.
For e.g. for French it becomes `https://ethereum.org/fr/assets/svgs/eth-diamond-glyph.svg` and for Italian it becomes `https://ethereum.org/it/assets/svgs/eth-diamond-glyph.svg`. Notice `fr` and `it` gets added in the link. This problem is with all the svg links on the page.

**Comment by minimalsm** [comment]
@kamuik16 I think you're right.

The bug looks like it was introduced [in this commit](https://github.com/ethereum/ethereum-org-website/commit/e1bd78d673aef028ebb31372bea2566fb9e1124d) during the migration from Gatsby to NextJS.

For context, our English links before the migration had the `en` prefix (e.g., `https://ethereum.org/en/`), but during the migration, we removed them for English but kept them in all other languages (e.g., `https://ethereum.org` for English, `https://ethereum.org/fr/` for French).

In [this PR](https://github.com/ethereum/ethereum-org-website/pull/12134) (which was merged yesterday, and will go live soon), we re-added the `en` prefix for English, so this bug is now also present in English (see deploy preview from that PR: https://deploy-preview-12134--ethereumorg.netlify.app/en/assets/).

To solve this, I think we need to add the lang prefix to the src in the `AssetDownloadImage` component.

cc: @pettinarip @wackerow

**Comment by FaybianB** [comment]
@minimalsm May I be assigned this issue?

**Comment by minimalsm** [comment]
Hey @FaybianB thanks for offering to help! 

Sure, all yours :)!

**Comment by wackerow** [comment]
> @kamuik16 I think you're right.
> 
> The bug looks like it was introduced [in this commit](https://github.com/ethereum/ethereum-org-website/commit/e1bd78d673aef028ebb31372bea2566fb9e1124d) during the migration from Gatsby to NextJS.

The thing I see changed here is we're using a `StaticImageData` for our image... this is when we import an image as a component (ie. `import SomeImage from "./some/path"`)... we then use `const imgSrc = (image as StaticImageData).src` inside `AssetDownload` to get the "source path"... but this is not an accessible path for downloading the image (both png's or svg's).

To get a "source path" that links to a downloadable image we may need to manually add this as a string prop for `AssetDownload`... ie:

```tsx
import learnHero from "@/public/heroes/learn-hub-hero.png"

// ...

          <AssetDownload
            title={t("page-assets-learn-hero-name")}
            alt={t("page-assets-learn-hero-name")}
            image={learnHero}
            imgSrc="./heroes/learn-hub-hero.png"
            artistName="Liam Cobb"
            artistUrl="https://liamcobb.com/"
          />
```

...and then update `AssetDownload` to accept this prop and use it for the link `href`

@FaybianB Does that make sense? Using the `.src` property of a statically imported an image will break inside an `href`, but a string relative to the `public` directory should be available at `https://ethereum.org/path/file.ext` (for example, http://ethereum.org/heroes/learn-hub-hero.png)

**Comment by wackerow** [comment]
@pettinarip Not sure if you have a better suggestion here aside from just manually adding the src path string (relative to public) as a prop for `AssetDownload`

**Comment by minimalsm** [comment]
> Hmm. Looks like we've removed SVG download entirely, so this is no longer an issue.
> 
> Was this intended? Can I close this issue? @wackerow @corwintines

Fixed in #13726
